numpy/core/tests/test_ufunc.py

```diff
@@ -866,6 +920,89 @@ def test_innerwt_empty(self):
         w = np.array([], dtype='f8')
         assert_array_equal(umt.innerwt(a, b, w), np.sum(a*b*w, axis=-1))
 
+    def test_cross1d(self):
+        """Test with fixed-sized signature."""
+        a = np.eye(3)
+        assert_array_equal(umt.cross1d(a, a), np.zeros((3, 3)))
+        out = np.zeros((3, 3))
+        result = umt.cross1d(a[0], a, out)
+        assert_(result is out)
+        assert_array_equal(result, np.vstack((np.zeros(3), a[2], -a[1])))
+        assert_raises(ValueError, umt.cross1d, np.eye(4), np.eye(4))
+        assert_raises(ValueError, umt.cross1d, a, np.arange(4.))
+        assert_raises(ValueError, umt.cross1d, a, np.arange(3.), np.zeros((3, 4)))
+
+    def test_can_ignore_signature(self):
```

Comment:
Any way to split this 70-line "unit test" up into smaller pieces logically?

Reply:
This come from PR #11175 which must be merged before this one. @mhvk any thoughts?

Reply:
Yes, my fault. Might be good to refactor, but out of scope for this PR, I think.